roles/commcare_analytics/tasks/rabbitmq.yml

```diff
@@ -0,0 +1,98 @@
+---
+- name: Install Erlang dependency
+  apt:
+    name: erlang
+    state: present
+  tags: rabbitmq
+
+- name: Add trusted key
+  apt_key:
+    url: https://packagecloud.io/rabbitmq/rabbitmq-server/gpgkey
+    state: present
+  tags: rabbitmq
+
+- name: Add rabbitmq official apt repository
+  apt_repository:
+    repo: 'deb https://packagecloud.io/rabbitmq/rabbitmq-server/ubuntu {{ ansible_distribution_release|lower }} main'
+    state: present
+    update_cache: "{{ apt_update_cache }}"
+  tags: rabbitmq
+
+- name: Install package
+  apt:
+    name: "rabbitmq-server={{ rabbitmq_version }}"
+    update_cache: "{{ apt_update_cache }}"
+    state: present
+  ignore_errors: '{{ ansible_check_mode }}'
+  tags: rabbitmq
+
+- name: Place config file
+  template:
+    src: "{{ item.src }}"
+    dest: "/etc/rabbitmq/{{ item.dest }}"
+    group: root
+    owner: rabbitmq
+    mode: 0644
+  loop:
+    - {"src": "rabbitmq/rabbitmq.conf.j2", "dest": "rabbitmq.conf"}
```

Comment:
Since there is only one element in the loop, would it make sense to have static values for `source` and `dest` above?

Reply:
Ya, I held on to it from the cluster implementation I copied over and thought of retaining it for future.

https://github.com/dimagi/commcare-analytics-ansible/pull/44/commits/7aa447fa374e8373e9360518bcc370cd7d1c638a